graphql/query_resolver.go

```diff
@@ -238,7 +238,14 @@ func (r *queryResolver) HostEvents(ctx context.Context, hostID string, hostTag *
 	if h == nil {
 		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("host '%s' not found", hostID))
 	}
-	events, count, err := event.MostRecentPaginatedHostEvents(h.Id, h.Tag, *limit, *page, false)
+	hostQueryOpts := event.MostRecentPaginatedHostEventsOpts{
```

Comment:
Maybe for another ticket but hostTag param is unused here it looks like

Reply:
it's probably okay since in the front end PR I will switch to using `host.Events` instead of `query.hostEvents` (this one)!